cli/client/dcoscli_test.go

```diff
@@ -0,0 +1,63 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type DcosCliTestSuite struct {
+	suite.Suite
+}
+func TestDcosCliTestSuite(t *testing.T) {
+	suite.Run(t, new(DcosCliTestSuite))
+}
+
+func (suite *DcosCliTestSuite) TestGetCLIValues() {
+	cachedConfig = nil
+
+	config, err := cliDiskConfig("testdata/config_missing")
+	assert.Equal(suite.T(), "open testdata/config_missing/.dcos/clusters: no such file or directory", err.Error())
```

Comment:
I'd just assert that there is an error.

I'm not a big fan of having to update text in multiple places.

Reply:
Or just assert containment or some such.